### PR TITLE
[plugin.video.newyankeeworkshop] 0.0.3

### PR DESCRIPTION
**Pull request by th0ma5w**
### Description
Updated code to Python 3

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by th0ma5w** [comment]
After studying this some more, perhaps this should be a Matrix-only commit... let me know and I can resubmit this on that branch only, or is Krypton expected to also receive Python 3?

File: plugin.video.newyankeeworkshop/addon.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.newyankeeworkshop" name="The New Yankee Workshop" version="0.0.2" provider-name="th0ma5w">
+<addon id="plugin.video.newyankeeworkshop" name="The New Yankee Workshop" version="0.0.3" provider-name="th0ma5w">
 	<requires>
-		<import addon="xbmc.python" version="2.25.0" />
+		<import addon="xbmc.python" version="3.0.0"/>
```
Comment (enen92) [inline plugin.video.newyankeeworkshop/addon.xml:4]: Can't do this. Addons with xbmc.python 3.0.0 should be submitted to the matrix branch

**Review by enen92 — CHANGES_REQUESTED** [review]
Please keep this one as is and submit a new version to matrix.
Thanks

**Comment by th0ma5w** [comment]
Will submit a new pull request against the Matrix branch

**Pull request closed without merging**